clusterloader2/testing/experimental/storage/pod-startup/volume-types/persistentvolume/pvc.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: {{.Name}}
+  labels:
+    app: {{.Name}}
+    group: {{.Group}}
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: {{.VolSize}}
+  persistentVolumeReclaimPolicy: Delete
+  volumeBindingMode: Immediate
```

Comment:
nit: add end-of line

Reply:
Reclaim policy and binding mode are fields on the storageclass, not PVC.

Until we have a way to measure and subtract out provisioning time, a prerequisite of running this test using default storageclass is that the default storageclass needs to have binding mode = immediate.

Reply:
I removed those two lines and added an assumption to the top.